rotatingtext/src/main/java/com/sdsmdg/harjot/rotatingtext/RotatingTextSwitcher.java

```diff
@@ -32,13 +37,14 @@
     private Rotatable rotatable;
 
     private Paint paint;
-
+    static int n = 0,val = 0;
```

Comment:
Can you please suggest some way for me to avoid using static

Reply:
Just check if it can be done using the instance variables. My concern is related to possible memory leaks while using static variables. https://stackoverflow.com/a/19824882/9063769.

Also, rename them to something more intuitive.

Reply:
@dev-ritik 
I am not able to find a way to use instance variables ...the main problem being whenever the functions of this (RotatingTextSwitcher) class get called the values of n and val get reinitialized to zero. 
Any suggestions?

Reply:
@dev-ritik  
Yes it worked thanks a lot!
Please review my code ,thanks!

Reply:
Cool great!:+1: